README.md

```diff
@@ -89,7 +89,7 @@ Examples of `Build` resource using the example strategies installed by default.
   $ hack/install-tekton.sh
   ```
 
-* Install Shipwright and sample strategies via `make`:
+* Install the operator and sample strategies via `make`:
```

Comment:
Given the current discussion with regards to naming, I would leave it as `Shipwright` for now.